docs/source/usage.rst

```diff
@@ -172,6 +172,54 @@ download the available results.
 
 ..  figure:: ./images/use/download-modal.png
 
+Possible error
+~~~~~~~~~~~~~~~~
+
+You may get the following error if you try to run libertem-server for a second time
+
+.. code-block:: shell
+
+    OSError: [WinError 10048] Only one usage of each socket address (protocol/network address/port) is normally permitted
```

Comment:
What is the root cause of this error? Is it reproducible? If so, how? Instead of documenting it here, we should strive to fix the cause instead.

Reply:
I think it is because of some problem in starting the server. I came across [this](https://stackoverflow.com/questions/12362542/python-server-only-one-usage-of-each-socket-address-is-normally-permitted) and it appears that we need to shut down the current process in order to start the next process. You can reproduce this by making the same changes in `win_tweaks.py` file as I have made.

Reply:
> down the current process in order to start the next process

Yes, but what was the reason the current process was not shut down? This should not be caused by the changes you did in this PR.

Reply:
This happens only when libertem-server is run from Bash. Everything's alright with Anaconda Prompt.

Reply:
When specifically does this happen? Does this also happen without your changes? If so, please log a separate issue for this, with all details on how to reproduce this, so we can fix this. The fix should not be part of this PR: each PR should only be a single logical change, if possible at all.

Reply:
> 
> 
> When specifically does this happen? 

At first the libertem-server has to be run from Bash. Then, both the Bash console and webpage have to be closed. Now, if you try to run the server again (either from Bash or Anaconda prompt), you will get the error. If we start the server from Anaconda at first, there will not be any error.

> 
> 
> Does this also happen without your changes? 

 No, because without my changes the server won't start at all from Bash.

>
> If so, please log a separate issue for this, with all details on how to reproduce this, so we can fix this. The fix should not be part of this PR: each PR should only be a single logical change, if possible at all.

Reply:
> Then, both the Bash console and webpage have to be closed

So you directly close the bash console using the "X" GUI element, without stopping the libertem-server before? If so, does this also happen if you close libertem-server using Ctrl+C before closing the bash console?

> No, because without my changes the server won't start at all from Bash.

￼Right, thanks. Please do log this *as a separate issue*, which can then be fixed after this PR is merged.

Reply:
> 
> So you directly close the bash console using the "X" GUI element, without stopping the libertem-server before? 

Yes, happens only when the 'X' GUI element is used. 


> 
> If so, does this also happen if you close libertem-server using Ctrl+C before closing the bash console?
 
No, doesn't happen with Ctrl+C

Reply:
:+1: that sounds like something that should be fixed.